frontend/src/pages/SignupPage/index.js

```diff
@@ -158,27 +158,22 @@ const SignupPage = () => {
       ),
     },
     STEP_4: {
+      buttonText: "I agree",
       component: (
         <PolicyForm handleChange={handleChange} submitData={submitData} />
       ),
     },
   };
 
+  let buttonHandler = state.step === STEP_4 ? submitData : continueToNextStep;
```

Comment:
Why not `const`?

Reply:
Oh right, João explained me better the difference between the two. I'll change it!